phfs/phoenixd.c

```diff
@@ -75,8 +92,12 @@ int phoenixd_open(const char *file, unsigned int major, unsigned int minor, unsi
 		return -EIO;
 	else if (msg_getlen(&rmsg) != sizeof(u32))
 		return -EIO;
-	else if (!(fd = *(u32 *)rmsg.data))
-		return -EIO;
+	else {
```

Comment:
Pls add brackets to previous conditions too

Reply:
Both done in the force-push